docs/automodapi/__init__.py

```diff
@@ -0,0 +1,18 @@
+# The code here was adapted from v0.14.0.dev0 of sphinx_automodapi
```

Comment:
Should note the license of automodapi here! Also, dev0 is probably autogenerated and it'd be better to point to a particular commit.

Reply:
There is still a lot of clean up to be had here and there, including properly linking to `sphinx_automodapi`.  I just want to get the bulk working and then I'll go back to dot the i's and cross the t's.